### PR TITLE
Update overview to include table64

### PR DESCRIPTION
**Pull request by sbc100**
See #51

### CODE REVIEW

File: proposals/memory64/Overview.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ This page describes a proposal to support linear memory of sizes larger than
 2<sup>32</sup> bits. It provides no new instructions, but instead extends the
 currently existing instructions to allow 64-bit indexes.
 
+In addition, in order to support source languages with 64-bit pointer width,
+this proposal also extends indirect function tables to allow 64-bit indexes.
```
Comment (dschuff) [inline proposals/memory64/Overview.md]:
```suggestion
this proposal also extends tables to allow 64-bit indexes.
```

Comment (dschuff) [reply proposals/memory64/Overview.md]: I assume we want this for all tables, not just tables containing funcref.

Comment (sbc100) [reply proposals/memory64/Overview.md]: Indeed.  Done.

File: proposals/memory64/Overview.md [hunk 2]
```diff
@@ -151,6 +164,56 @@ have to support 32-bit memory addresses in their ABI.
...
+    - ```
+        C.tables[x] = it limits t
+      ------------------------------
+      C ⊦ table.set x : [it] → [t]
+      ```
+  - table.size x
+    - ```
+        C.tables[x] = it limits t
+      ------------------------------
+      C ⊦ table.size x : [it] → [it]
```
Comment (rossberg) [inline proposals/memory64/Overview.md]:
```suggestion
      C ⊦ table.size x : [] → [it]
```

**Review by rossberg — APPROVED** [review]
LGTM modulo comment

**Pull request merged**